crates/ui/src/components/title_bar/windows_window_controls.rs

```diff
@@ -94,6 +94,25 @@ impl WindowsCaptionButton {
             hover_background_color,
         }
     }
+
+    #[cfg(not(target_os = "windows"))]
+    fn get_font() -> &'static str {
+        "Segoe Fluent Icons"
+    }
+
+    #[cfg(target_os = "windows")]
+    fn get_font() -> &'static str {
+        use windows::Wdk::System::SystemServices::RtlGetVersion;
+
+        let mut version = unsafe { std::mem::zeroed() };
+        let status = unsafe { RtlGetVersion(&mut version) };
```

Comment:
What's the cost of calling this every render?

Reply:
Unsure on that but made a change to make it better, it seems to run fine.

Reply:
That change you made didn't actually change anything, as the constructor is still called on each render.

We can leave it the way it was, so long as it's not expensive to call potentially on every frame.